app/src/main/java/com/example/imagefilter/ui/mainScreen/component/AppRangeSlider.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.imagefilter.ui.mainScreen.component
+
+import androidx.compose.material.Slider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
```

Comment:
Nit - We can mark this composable with @NonRestartableComposable as this does not have a mechanism of its own. It just takes in values and passes them down to another composable